### PR TITLE
Bio-Formats 5.9.1 release

### PR DESCRIPTION
**Pull request by dgault**
See https://trello.com/c/sLkiWlua/24-open-pr-against-wwwopenmicroscopyorg

Also includes the updates to contributors from https://github.com/openmicroscopy/bioformats/pull/3178

### CODE REVIEW

**Comment by dgault** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/08/14/bio-formats-5-9-1.html

**Comment by dgault** [comment]
Not sure why the failure is occurring as the page appears to present:

`internally linking to /2018/08/14/bio-formats-5-9-1.html, which does not exist (line 86)`

**Comment by sbesson** [comment]
Might be related to #242 i.e. the production configuration currently disable future posts.

**Comment by sbesson** [comment]
https://travis-ci.org/openmicroscopy/www.openmicroscopy.org/builds/414195485 now green with the changes from #242 merged. Will list for a final content review on Monday.

**Comment by sbesson** [comment]
Looks great. Thanks @dgault. Merging and tagging for public release on Tuesday.

File: _posts/2018-08-14-bio-formats-5-9-1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+Today we are releasing Bio-Formats 5.9.1 which includes the following changes:
+
+File format fixes and improvements:
+
+* Olympus OIR
+    * fixed a bug to prevent the incorrect file from being read when multiple datasets are in the same location
+* LEO
+    * updated parsing of metadata values for image pixel size, working distance, filament, EHT and date(thanks to David Mankus)
```
Comment (mtbc) [inline _posts/2018-08-14-bio-formats-5-9-1.md]: space before "("

File: _posts/2018-08-14-bio-formats-5-9-1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Deltavision
```
Comment (mtbc) [inline _posts/2018-08-14-bio-formats-5-9-1.md]: DeltaVision

File: _posts/2018-08-14-bio-formats-5-9-1.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    * reader can now detect up to 12 channels
+* Micro-Manager
+    * added a warning to the logging when an image is acquired with an unsupported version
+
+Documentation improvements:
+
+* added QuPath to the list of visualization and analysis applications
+* updated the link to the i3dcore library
+* updated the link to Slidebook
+* improved MATLAB documentation with information on java heap memory preferences (thanks to Kouichi C. Nakamura)
```
Comment (mtbc) [inline _posts/2018-08-14-bio-formats-5-9-1.md]: Java

**Comment by mtbc** [comment]
At @sbesson's suggestion opened https://github.com/openmicroscopy/www.openmicroscopy.org/pull/247.

**Pull request merged**